Kiosk/SettingsHelper.cs

```diff
@@ -145,7 +146,7 @@ public static SettingsHelper Instance
             }
         }
 
-        private async void LoadRoamingSettings()
+        private void LoadFaceRoamingSettings()
```

Comment:
Let's keep all settings inside the LoadRoamingSettings function.

Reply:
Yes, check line 385 in the same file

Reply:
Yes, I've seen your logic for splitting these roaming settings into multiple functions. But I think we can keep all these settings inside one function.

Reply:
I do this because for my was crazy check where change anomaly settings and compare with other service, probably other contributor could determine if you see well

Reply:
I actually heard that comment before. That function started small, but have certainly grown very long overtime. Let's go ahead with breaking it up. Thanks for the help with this. Perhaps once they are broken down like this we can explore the similarities between the various functions and think of a way to create a common parameterized function.

Reply:
All you comments fixed, sorry for extra blank lines and sure that + concatenations have a better way. I also think that these functions can be even more parameterized